templates/base/cluster-template.yaml

```diff
@@ -160,6 +166,8 @@ spec:
                     value: "${KUBEVIP_SVC_ENABLE=false}"
                   - name: lb_enable
                     value: "${KUBEVIP_LB_ENABLE=false}"
+                  - name: enableServicesElection
```

Comment:
why is this required?

Reply:
this is for `Allow to enable Load Balancing SVC Load Balancers`